setup.py

```diff
@@ -133,6 +133,7 @@ def get_git_sha():
     classifiers=[
         "Programming Language :: Python :: 3.6",
```

Comment:
Should we remove this line?

Reply:
Let's keep it there a while longer and drop it in a separate PR where `dataclasses` and other backwards compatibility libs are also dropped.